packages/babel-plugin-transform-react-jsx-development/test/fixtures/linux/fragments/output.js

```diff
@@ -6,4 +6,4 @@ var x = /*#__PURE__*/_reactJsxDevRuntime.jsxDEV(_reactJsxDevRuntime.Fragment, {
     lineNumber: 1,
     columnNumber: 11
   }, this)
-}, void 0, false);
+}, void 0, false, void 0, void 0);
```

Comment:
Can we avoid emitting these two? By default terser can not strip them because otherwise `arguments` length will be changed.